client/components/Facilitator/Components/Cohorts/CohortDataTableMenu.jsx

```diff
@@ -34,14 +35,16 @@ export class CohortDataTableMenu extends React.Component {
 
         return (
             <Menu icon>
-                <Popup
-                    content="Close this data table tab"
-                    trigger={
-                        <Menu.Item name="close" onClick={onClick}>
-                            <Icon name="close" />
-                        </Menu.Item>
-                    }
-                />
+                <ConfirmAuth requiredPermission="edit_scenarios_in_cohort">
+                    <Popup
+                        content="Close this data table tab"
+                        trigger={
+                            <Menu.Item name="close" onClick={onClick}>
+                                <Icon name="close" />
+                            </Menu.Item>
+                        }
+                    />
+                </ConfirmAuth>
```

Comment:
The close button doesn't make a lot of sense in the context of the participant data view

Reply:
Brilliant